transformer_engine/pytorch/module/base.py

```diff
@@ -116,35 +130,22 @@ def initialize_ub(
         intra_node_group = world_group
```

Comment:
Does this line redundant?

Reply:
Line 133 was a typo. It's meant to be setting `intra_node_ranks = list(range(world_size))`. Thanks for catching that!